set.mm

```diff
@@ -22321,19 +22341,34 @@ of the unique existential quantifier (note that ` y ` and ` z ` need not
 
   ${
     $d x y z $.  $d y z ph $.
+    $( Proof lines shared by ~ eu6 and ~ eu6-1 .  (Contributed by NM,
+       12-Aug-1993.)  This used to be the definition of the unique existential
+       quantifier, while ~ df-eu was then proved as ~ dfeu .  (Revised by BJ,
+       30-Sep-2022.)  (Proof shortened by Wolf Lammen, 3-Jan-2023.)  Extract
+       common proof lines.  (Revised by Wolf Lammen, 3-Mar-2023.) $)
+    eu6lem $p |- ( E. y A. x ( ph <-> x = y ) <->
+          ( E. y A. x ( x = y -> ph ) /\ E. z A. x ( ph -> x = z ) ) ) $=
+      ( weq wb wal wex wi wa 19.42v pm3.33 alanimi equvelv sylib pm4.71i albiim
+      biancomi equequ2 exbii imbi2d albidv anbi2d syl5bb pm5.32ri bitr4i ax6evr
+      biantru 3bitr4ri exdistrv bitri ) ABCEZFBGZCHULAIZBGZABDEZIZBGZJZDHZCHUOC
+      HURDHJUMUTCUMCDEZJZDHUMVADHZJUTUMUMVADKUSVBDUSUSVAJVBUSVAUSULUPIZBGVAUNUQ
+      VDBULAUPLMCDBNOPVAUMUSUMUOAULIZBGZJVAUSUMUOVFAULBQRVAVFURUOVAVEUQBVAULUPA
+      CDBSUAUBUCUDUEUFTVCUMDCUGUHUITUOURCDUJUK $.
+
     $( Alternate definition of the unique existential quantifier ~ df-eu not
        using the at-most-one quantifier.  (Contributed by NM, 12-Aug-1993.)
        This used to be the definition of the unique existential quantifier,
        while ~ df-eu was then proved as ~ dfeu .  (Revised by BJ, 30-Sep-2022.)
        (Proof shortened by Wolf Lammen, 3-Jan-2023.) $)
     eu6 $p |- ( E! x ph <-> E. y A. x ( ph <-> x = y ) ) $=
-      ( vz weu wex weq wi wal wa wb eu3v exdistrv 19.42v pm3.33 alanimi equvelv
-      sylib exbii 3bitr4ri pm4.71i albiim biancomi equequ2 imbi2d albidv anbi2d
-      syl5bb pm5.32ri bitr4i ax6evr biantru exsb anbi1i bitri ) ABEABFZABDGZHZB
-      IZDFZJZABCGZKBIZCFZABDLVBAHZBIZUSJZDFZCFVFCFZUTJVDVAVFUSCDMVCVHCVCCDGZJZD
-      FVCVJDFZJVHVCVCVJDNVGVKDVGVGVJJVKVGVJVGVBUQHZBIVJVEURVMBVBAUQOPCDBQRUAVJV
-      CVGVCVFAVBHZBIZJVJVGVCVFVOAVBBUBUCVJVOUSVFVJVNURBVJVBUQACDBUDUEUFUGUHUIUJ
-      SVLVCDCUKULTSUPVIUTABCUMUNTUO $.
+      ( vz wex weq wi wal wa weu wb exsb anbi1i eu3v eu6lem 3bitr4i ) ABEZABDFG
+      BHDEZIBCFZAGBHCEZRIABJASKBHCEQTRABCLMABDNABCDOP $.
+
+    $( One direction of ~ eu6 needs fewer axioms.  (Contributed by Wolf Lammen,
+       2-Mar-2023.) $)
+    eu6-1 $p |- ( E. y A. x ( ph <-> x = y ) -> E! x ph ) $=
+      ( vz weq wb wal wex wi wa weu eu6lem exsb-1 anim1i sylbi eu3v sylibr ) AB
+      CEZFBGCHZABHZABDEIBGDHZJZABKSRAIBGCHZUAJUBABCDLUCTUAABCMNOABDPQ $.
```

Comment:
couldn't you prove eu6 from eu6-1, and inline/remove eu6lem ?

Reply:
You could ask the same question for exsb.  For this to have any effect, the proof should be based on sth like cb3v2.  Unfortunately you need the role of ph and ps reversed there, which finally drags in all of ax-10, ax-11 and ax-12 again.  I gave up after a few tries, and since the net effect of separating the directions is just extra complexity, so I abandoned that thought.